Web/Presenters/GroupPresenter.php

```diff
@@ -83,9 +83,23 @@ function renderFollowers(int $id): void
     {
         $this->assertUserLoggedIn();
         
-        $this->template->club      = $this->clubs->get($id);
-        $this->template->followers = $this->template->club->getFollowers((int) ($this->queryParam("p") ?? 1));
-        $this->template->count     = $this->template->club->getFollowersCount();
+        $this->template->club              = $this->clubs->get($id);
+        $this->template->onlyShowManagers  = $this->queryParam("onlyAdmins") == "1";
+        if($this->template->onlyShowManagers) {
+            $this->template->followers     = null;
+
+            $this->template->managers     = $this->template->club->getManagers((int) ($this->queryParam("p") ?? 1), !$this->template->club->canBeModifiedBy($this->user->identity));
+            if($this->template->club->canBeModifiedBy($this->user->identity) || !$this->template->club->isOwnerHidden()) {
+                $this->template->managers  = array_merge(array($this->template->club->getOwner()), iterator_to_array($this->template->managers));
+            }
+
+            $this->template->count         = $this->template->club->getManagersCount();
+        } else {
```

Comment:
Instead of else you can return early here.

Reply:
Done

Reply:
Done